cluster/gce/util.sh

```diff
@@ -57,23 +59,43 @@ fi
 
 # Sets node image based on the specified os distro. Currently this function only
 # supports gci and debian.
-function set-node-image() {
-    if [[ "${NODE_OS_DISTRIBUTION}" == "gci" ]]; then
+function set-linux-node-image() {
+    if [[ "${LINUX_NODE_OS_DISTRIBUTION}" == "gci" ]]; then
         DEFAULT_GCI_PROJECT=google-containers
         if [[ "${GCI_VERSION}" == "cos"* ]]; then
             DEFAULT_GCI_PROJECT=cos-cloud
         fi
 
         # If the node image is not set, we use the latest GCI image.
         # Otherwise, we respect whatever is set by the user.
-        NODE_IMAGE=${KUBE_GCE_NODE_IMAGE:-${GCI_VERSION}}
-        NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-${DEFAULT_GCI_PROJECT}}
+        LINUX_NODE_IMAGE=${KUBE_GCE_NODE_IMAGE:-${GCI_VERSION}}
+        LINUX_NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-${DEFAULT_GCI_PROJECT}}
+    fi
+}
+
+# Requires:
+#   WINDOWS_NODE_OS_DISTRIBUTION
+# Sets:
+#   WINDOWS_NODE_IMAGE_FAMILY
+#   WINDOWS_NODE_IMAGE_PROJECT
+function set-windows-node-image() {
+    WINDOWS_NODE_IMAGE_PROJECT="windows-cloud"
+    if [[ "${WINDOWS_NODE_OS_DISTRIBUTION}" == "win1803" ]]; then
```

Comment:
Are we also creating `win1809` and `win2019` directories in `gce/`?

Reply:
We may or may not need those directories later. I'm not sure to what extent we'll need different code or constants for different Windows versions. I'm ok with just leaving the directory as 'win1803' for now.